test/Function/StackArgsWithFormals.js

```diff
@@ -0,0 +1,141 @@
+//Test1 : Tests Optimization with bailout.

```

Comment:
Done.

---
In reply to: [67765651](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67765651) [](ancestors = 67765651)